backend/src/manager/mod.rs

```diff
@@ -438,6 +440,9 @@ async fn manager_thread_loop(mut recv: Receiver<OnStop>, thread_shared: &Arc<Man
                     }
                 }
                 tracing::error!("service crashed: {}: {}", e.0, e.1);
+                if &e.1 == KILLED && e.0 == 137 {
+                    break;
```

Comment:
If there was our killed by the kill signal, expecting that we are either killing or exiting so we shouldn't continue on this loop